dib/plan.go

```diff
@@ -82,6 +102,22 @@ func checkDiff(graph *dag.DAG, diffs []string) {
 	}
 }
 
+func MatchPattern(node *dag.Node, file string) bool {
```

Comment:
No need to export since it is not used from an other package
```suggestion
func matchPattern(node *dag.Node, file string) bool {
```